### PR TITLE
doc: update list of contributors

### PR DESCRIPTION
**Pull request by monperrus**
Spoon is great only thanks to our awesome contributors!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,52 +149,77 @@ Spoon is Free and Open Source, double-licensed under the ([CeCILL-C license](htt
...
 * Filip Krakowski
 * Lukas Krejci
+* Rohitesh Kumar Jain
 * Ashutosh Kumar Verma
+* Simon Larsén
+* Quentin LE DILAVREC
 * Roman Leventov
 * Fan Long
 * Kai Luo
-* Matias Martinez
```
Comment (slarse) [inline README.md:178]: The one-liner to extract names assumes that everyone has configured `user.name` to a "proper" name of more than two words. This causes a failure to find for example @martinezmatias here. And @MartinWitt, and @andrewbwogi, and many others. This is why I think may be better to just let GitHub figure out who the contributors are, and link to the contributors page like @algomaster99 did here: https://github.com/spoonlabs/diffmin

Here's a full list of single-word names that are listed in the commit history as contributors, and thus will be overlooked by the one-liner:

* adamjryan
* andrewbwogi
* anonsvn
* argius
* arno
* arno_b
* Artamm
* aryan
* aveuiller
* bcornu
* bdanglot
* Charm
* ChrisSquare
* danglotb
* dufaux
* dwayneb
* dya-tel
* Egor18
* fangzhen
* fav
* Gerard
* GerardPaligot
* gibahjoe
* GluckZhang
* HectorSM
* jakobbraun
* jon
* leventov
* lodart
* Marcel
* martin
* martinezmatias
* MartinWitt
* matiasmartinez
* maxcleme
* noguera
* peroksid90
* petitpre
* raymogg
* renaud
* santos-samuel
* scootafew
* seinturier
* Shantanu
* srlm
* ST0NEWALL
* tdurieux
* tiagodrcarvalho
* Wreulicke
* Дмитрий

Comment (slarse) [reply README.md:178]: And also, I think it feels worse as a contributor to not be included in an existing list of contributors, than there not being a list to begin with. Just my two cents, make of it what you will! :)

**Comment by monperrus** [comment]
This list is a symbol, we want to acknowledge all contributors in a special manner, with more than the default Github page that any random repo has.

But, you're right @slarse, we can do much better than the bash one-liner, see new file `chore/generate-contributor-list.py`

**Comment by nharrand** [comment]
LGTM!

**Pull request merged**